auto_install/install.sh

```diff
@@ -1055,10 +1055,11 @@ confOVPN() {
     # verify server name to strengthen security
     $SUDO sed -i "s/SRVRNAME/${SERVER_NAME}/" /etc/openvpn/easy-rsa/pki/Default.txt
 
-    if [ ! -d "/home/$pivpnUser/ovpns" ]; then
-        $SUDO mkdir "/home/$pivpnUser/ovpns"
+    INSTALL_HOME=$(cat /etc/passwd | grep "$INSTALL_USER" | cut -d: -f6)
```

Comment:
@xptsp 
`cat file | grep` is btw an unnecessary doubled command use. `grep` is actually made to search through files:
```
grep "^${INSTALL_USER}:" /etc/passwd | cut -d: -f6)
```
- Additionally I wrapped the user name variable into line start `^` and `:` to assure it does not match any other line that contains the user name string as e.g. default group, home dir or login shell, or e.g. another user that has the same with prefix or suffix.